test/runtests.jl

```diff
@@ -1,11 +1,11 @@
 import ProgressMeter
 using Test
 
-include("core.jl")
-include("test.jl")
-include("test_showvalues.jl")
+# include("core.jl")
+# include("test.jl")
+# include("test_showvalues.jl")
 include("test_map.jl")
-include("test_float.jl")
+# include("test_float.jl")
```

Comment:
Committed these by accident?

Reply:
Yes! :man_facepalming: sorry! Is there a way to only run certain test sets so that I don't have to comment out ones to skip?

Reply:
So what's the plan to fix it? :)

Reply:
@zsunberg, if you want to just run that one test you can just navigate to the test directory and say `include("test_map.jl")`.

Ideally, this should have been fixed when it was first pointed out. Fixed in 237365084b3fd04bd1c8446d5dd225ebf6be86df

Reply:
Yes, I apologize! I am moving to a new state this week, so it slipped through the cracks.